Cython/Utility/CppConvert.pyx

```diff
@@ -76,7 +76,7 @@ cdef object {{cname}}(const vector[X]& v):
     for i in range(v_size_signed):
```

Comment:
This'll get broken again by https://github.com/cython/cython/pull/6077 fairly soon. Not your problem necessarily though (I can go fix it up after)

Reply:
Fix will be https://github.com/cython/cython/pull/6077

Reply:
Thanks!